pkg/schedule/checker/replica_checker.go

```diff
@@ -67,40 +71,40 @@ func (*ReplicaChecker) GetType() types.CheckerSchedulerType {
 }
 
 // Check verifies a region's replicas, creating an operator.Operator if need.
-func (r *ReplicaChecker) Check(region *core.RegionInfo) *operator.Operator {
+func (c *ReplicaChecker) Check(region *core.RegionInfo) *operator.Operator {
```

Comment:
Does it seem to contain the change from another pr?

Reply:
No, this change is needed since we use r as the naming of the rand field.